src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -769,6 +769,12 @@ protected DynamoViewModel(StartConfiguration startConfiguration)
             }
 
             FileTrustViewModel = new FileTrustWarningViewModel();
+            model.ShutdownStarted += Model_ShutdownStarted;
```

Comment:
Isn't there already an event about model disposal or shutdown handling? I was hoping DynamoViewMode -> Dispose -> GuideManager-> Dispose-> Make sure Window resources close

Reply:
Probably you were looking for something like the WorkspaceViewModel's Dispose implementation in the DynamoViewModel, but there is not exist.

![image](https://github.com/DynamoDS/Dynamo/assets/96534278/ff06e463-f8d5-432d-b5eb-3cd899f7e858)

![image](https://github.com/DynamoDS/Dynamo/assets/96534278/ac6b22cb-e274-4a34-a080-2219c574fd28)

Reply:
it sounds clean, Done.